libs/ddd-angular/src/generators/api/generator.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import { Tree, readProjectConfiguration, getProjects } from '@nrwl/devkit';
+
+import domainGenerator from '../domain/generator';
+import generator from './generator';
+import { DddApiGeneratorSchema } from './schema';
+import {
+  changeIs,
+  generalProjectFiles,
+  generalTestingFiles,
+  nxFiles,
+} from '../../helpers/test-helper';
+import { projectGraphAdapter } from 'nx/src/project-graph/project-graph';
+
+describe('api generator', () => {
+  let appTree: Tree;
+  const options: DddApiGeneratorSchema = {
+    name: 'test',
+    domain: 'testing-area',
+  };
+
+  beforeEach(async () => {
+    appTree = createTreeWithEmptyWorkspace();
+    await domainGenerator(appTree, { name: options.domain });
+    await generator(appTree, {
+      name: options.name,
+      domain: `${options.domain}-domain`,
```

Comment:
${options.domain}-domain 
should be obtain by the only other project that exist

Reply:
Since we've parameterized the setup function, the generator takes it's options from the parameters which contain the domain project's name. Do you think it's still necessary? @andy2great